examples/all-clusters-app/all-clusters-common/gen/endpoint_config.h

```diff
@@ -356,7 +356,7 @@
 #define ATTRIBUTE_SINGLETONS_SIZE (6)
 
 // Total size of attribute storage
-#define ATTRIBUTE_MAX_SIZE 150
+#define ATTRIBUTE_MAX_SIZE 155
```

Comment:
No, 150 is the right number here as far as I can tell from adding up the actual attribute sizes.  I'll comment in https://github.com/project-chip/connectedhomeip/issues/4371 regarding what I think is going on.

Reply:
How about reverting only this but keeping updated code that checks for out of bounds and returns an error?

Also sanitizer logic is probably ok so I would like to keep it.

Reply:
@bzbarsky-apple - reverted this change. Running all clusters on linux will have a LOT of:

```
CHIP:ZCL: READ: attribute size too large for caller
CHIP:ZCL: ERR: reading cluster 0x 6 attribute 0x 0: 0x89
```

however it will not access invalid memory anymore.